draft-ietf-quic-ack-frequency.md

```diff
@@ -479,8 +502,14 @@ new ACK_FREQUENCY frame immediately upon confirmation of connection migration.
 
 
 # Security Considerations
+
 TBD.
 
+An improperly configured or malicious sender could cause standards-compliant receivers to
+acknowledge more frequently than their available resources permit, thus causing a
+denial-of-service (DoS) attack. However, a sender has control of many aspects of
+the receiver behaviour and therefore, for a well-configured sender, this introduces no additional threat.
```

Comment:
This text is a bit confusing in the use of "sender". However, I think the real advise is that a host should not just blindly apply the max ack delay value provided by the other end but may use a higher value if the available resources need to be protected. However, each endpoint actually indicates its minimum value in the transport parameter and this should protect this case. But this I guess there can be cases where you want to dynamically adopt, e.g. when you are under DoS...

The other side of this is, however, that a host that requests a certain max delay threshold can not rely on the fact that this threshold is always used. Which is probably also something important to note (and raised by Bob in issue #125). However, not sure if this should be in the security considerations section of somewhere else.

Reply:
/sender/ seems to be the sender of the ACK Frequency Frame.

Reply:
I think there is a possibility that we don't want to say much more: because QUIC frames are already protected from on-path attack. In that case, we are simply talking about a rogue sender of an ACK Frequency Frame to make a receiver do more work,... there are a lot more crazy things a sender of the ACK Frequency Frame and others QUIC Frames **could** do to mess up a receiver as a DoS attack, and it seems to me also there is no additional threat.